### ISSUE TITLE
Sphinx warning: node class bibliography is already registered

### ISSUE THREAD

**Issue by jamesjer** [mcmtroffaes/sphinxcontrib-bibtex#157]
I just attempted to build the 0.4.1 release for the Fedora Linux distribution.  Almost every test failed as follows:

```
ERROR: test_custom_style.test_custom_style
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/usr/lib/python3.7/site-packages/sphinx_testing/util.py", line 159, in decorator
    app = TestApp(*self.sphinxargs, **sphinxkwargs)
  File "/usr/lib/python3.7/site-packages/sphinx_testing/util.py", line 97, in __init__
    verbosity, parallel)
  File "/usr/lib/python3.7/site-packages/sphinx/application.py", line 191, in __init__
    self.setup_extension(extension)
  File "/usr/lib/python3.7/site-packages/sphinx/application.py", line 411, in setup_extension
    self.registry.load_extension(self, extname)
  File "/usr/lib/python3.7/site-packages/sphinx/registry.py", line 326, in load_extension
    metadata = mod.setup(app)
  File "/builddir/build/BUILD/sphinxcontrib-bibtex-0.4.1/sphinxcontrib/bibtex/__init__.py", line 145, in setup
    app.add_node(bibliography)
  File "/usr/lib/python3.7/site-packages/sphinx/application.py", line 483, in add_node
    type='app', subtype='add_node')
  File "/usr/lib64/python3.7/logging/__init__.py", line 1686, in warning
    self.log(WARNING, msg, *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/sphinx/util/logging.py", line 112, in log
    super(SphinxLoggerAdapter, self).log(level, msg, *args, **kwargs)
  File "/usr/lib64/python3.7/logging/__init__.py", line 1718, in log
    self.logger.log(level, msg, *args, **kwargs)
  File "/usr/lib64/python3.7/logging/__init__.py", line 1399, in log
    self._log(level, msg, args, **kwargs)
  File "/usr/lib64/python3.7/logging/__init__.py", line 1469, in _log
    self.handle(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 1479, in handle
    self.callHandlers(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 1541, in callHandlers
    hdlr.handle(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 851, in handle
    rv = self.filter(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 713, in filter
    result = f.filter(record)
  File "/usr/lib/python3.7/site-packages/sphinx/util/logging.py", line 382, in filter
    raise SphinxWarning(message)
sphinx.errors.SphinxWarning: while setting up extension ['?', 'sphinxcontrib.bibtex']: node class 'bibliography' is already registered, its visitors will be overridden
```

**Comment by jamesjer** [comment]
I should mention that this is with Sphinx 1.7.6.  Also, if I change this line in sphinxcontrib/bibtex/__init__py:
```
        app.add_node(bibliography)
```
to this:
```
        app.add_node(bibliography, override="True")
```
all of the tests pass.

**Comment by mcmtroffaes** [comment]
I cannot reproduce this locally for some reason (possibly, we're using a different testing framework?), however it should have been fixed in version 0.4.2, which was just released. If you still run into issues, please do let me know.
